docs/releases/2.7.0.rst

```diff
@@ -230,25 +252,76 @@ Infrastructure
-  removed in Pootle 2.5.0.
-- Live translation: Rarely enabled, and it was discouraged to use it.
-- Autosync: It was recommended to never use it. The files can now be synced
-  instead using the appropriate management commands.
+Recoverable features
+--------------------
+
+The following features are removed from Pootle since 2.5.1.1 and will be
+recovered at some time.  Where possible we provide alternate approaches that
+can be used.
 
-The voting feature for terminology suggestions has also been removed, due to
-its low popularity and high maintenance cost. In order to save existing voting
-data, make a backup of the ``voting_vote`` table before upgrading.
+.. note: sysadmins should take note of these changes and determine if this
+   prevents use of Pootle within their environment.  Essentially you will need
+   to evaluate the use and need for each missing feature.
+
+Major fatures
+~~~~~~~~~~~~~
+
+- Extension actions.
+- Tags and Goals.
+- Placeables support in the editor
+- SQLite support.
+- LDAP support.
+- Monolingual file format support - perform file conversion to and from
+  bilingual files outside of Pootle.
+- Support for Version Control Systems - automate your version control
+  integration outside of Pootle.
+- News, notifications and RSS feeds - make use of announcement pages use other
+  channels of communication.
+- Offline translation: file uploads and downloads.
+- Update against templates - do template updates outside of Pootle and use
+  :command:`update_stores` to load the changed files.
+- Public API.
+
+Minor features
+~~~~~~~~~~~~~~
+
+- The Wikipedia lookup backend
+- No *Top Contributors* tables - user scores likely provide the information you
+  are looking for.
+- Project/Language/Translation Project descriptions - these are migrated to
+  announcement static pages.
+- Management commands: update_translation_projects, updatedb, upgrade, setup
+- ``assign_permissions`` management command
+
+
+Legacy features
+---------------
+
+We have dropped these features, some of which have been kept around to allow
+easy upgrades in the past:
+
+.. note: The removal of some of these feature required extensive changes to the
+   upgrading code, which means that upgrading directly from very old Pootle
+   versions is no longer possible. In case you are trying to upgrade from
+   Pootle 2.1.0 or older you must first upgrade to 2.1.6 before continuing the
+   upgrade process.
+
+- .pending and .tm files support: Not necessary since the ``updatetm`` tool was
+  removed in Pootle 2.5.0.
+- Live translation: Rarely enabled, and its use was actively discouraged.
+- Autosync: It was recommended to never use it. The files can be synced using
+  :command:`sync_stores` instead.
+- The voting feature for terminology suggestions has also been removed, due to
+  its low popularity and high maintenance cost. In order to save existing
+  voting data, make a backup of the ``voting_vote`` table before upgrading.
```

Comment:
Please remove the phrase about backing up the `voting_vote` table. It is completely unnecessary to tell this on release notes since it only applies to users upgrading Pootle, and that is repeated a lot of times in the upgrade instructions already.